app/controllers/offers_controller.rb

```diff
@@ -115,6 +119,24 @@ def set_status
     status_setter(params)
   end
 
+  def clear_hris_status
+    params[:contracts].each do |id|
+      offer = Offer.find(id)
+      offer.update_attributes!(hris_status: nil)
+    end
+  end
+
+  def accept_offer
+    offer = Offer.find(params[:offer_id])
+    if offer[:status]!="Unsent"
+      offer.update_attributes!(status: "Accepted", accept_date: DateTime.now)
+      offer = offer.format
+      render status: 200, json: {message: "You've changed the status of #{offer[:applicant][:first_name]} #{offer[:applicant][:last_name]}'s offer for #{offer[:position]} to 'Accepted'."}
+    else
+      render status: 404, json: {message: "Error: You can't accept an Unsent offer."}
```

Comment:
I think that @jmzaleski wanted this to be a front-end confirmation, not something that is prevented in the back-end?